v1/ProberSpec.dhall

```diff
@@ -0,0 +1,3 @@
+{ Type = { url : Text, scheme : Optional Text, path : Optional Text }
+, default = { scheme = Some "http", path = Some "/probe" }
```

Comment:
Should be `None Text` for both `scheme`, `path` (if you want to document that `None Text` is equivalent to `http` or `/probe`, then put it in a comment)